### ISSUE TITLE
modified "connect" doesn't pass props

### ISSUE THREAD

**Issue by EyePulp** [kolodny/redux-await#4]
Howdy - I notice this module's wrapper of the redux "connect()" function doesn't pass the props through as the second argument, e.g.

``` javascript
// plain redux
import { connect }  from 'react-redux';
export default connect((state,props)=>{
  console.log('props:',props);  // has the original props being passed to the Container
  return state;
})(Container);

// redux-await
import { connect } from 'redux-await'; // it just spreads state.await on props
export default connect((state,props)=>{
  console.log('props:',props);  // undefined - no second arg
  return state;
})(Container);
```

I apologize, I wasn't totally confident in the patch I came up with, so I'm currently using the original redux function and manually adding the `state.await` values into the returned state.

Thanks for making this module!

**Comment by kolodny** [comment]
done here 4d1674e

I'm published it to `5.0.1` so you can just update it in npm. Let me know if that solves it

Thanks!

**Comment by EyePulp** [comment]
That took care of it, and it looks simpler than I thought it would be.  Thanks for issuing the patch so quickly!
